packages-next/fields/src/types/timestamp/views/index.tsx

```diff
@@ -11,50 +10,106 @@ import {
   FieldControllerConfig,
   FieldProps,
 } from '@keystone-next/types';
-import { jsx } from '@keystone-ui/core';
-import { FieldContainer, FieldLabel, TextInput } from '@keystone-ui/fields';
+import { jsx, Inline, Stack } from '@keystone-ui/core';
+import { FieldContainer, FieldLabel, TextInput, DatePicker, DateType } from '@keystone-ui/fields';
+import { TextInputProps } from '@keystone-ui/fields/src/TextInput';
 
 // TODO: Bring across the datetime/datetimeUtc interfaces, date picker, etc.
-
-function formatOutput(value: string) {
-  if (!value) return '';
-  const date = new Date(value);
-  return `${date.toLocaleDateString()} ${date.toLocaleTimeString()}`;
+interface TimePickerProps extends TextInputProps {
+  format: '12hr' | '24hr'
 }
 
+const TimePicker = ({ autoFocus, onBlur, disabled, onChange, format = '24hr', value }: TimePickerProps) => {
```

Comment:
very very rudimentary TimePicker component, currently just a input with some very light formatting on top. 
List of features to be added in JiRA